server/index-node/src/resolver.rs

```diff
@@ -100,9 +100,7 @@ where
 
         let block_hash = field
             .get_required::<H256>("blockHash")
-            .expect("Valid blockHash required")
-            .try_into()
-            .unwrap();
```

Comment:
Dumb question: I didn't understand how those two lines can be removed? 🤔

Reply:
I'm not sure what `try_into` turned the `H256` into, but `BlockPtr` has a `From<(H256, u64)>` impl.
`block_hash` is used to create `block` on the next line: `BlockPtr::from((block_hash, block_number))`